ui/push-health-view/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+// Vendor Styles
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'font-awesome/css/font-awesome.css';
+
+// Vendor JS
+import 'reactstrap';
```

Comment:
I think this `reactstrap` import can be removed, since (unlike the CSS) the JS is always imported at point of use.